app/models/news.rb

```diff
@@ -2,18 +2,19 @@
 class News < ActiveRecord::Base
   belongs_to :profile
 
-  has_attached_file :image, 
-                    :styles => { original: "4000x4000>", large: "800x800>", small: "250x250>",thumb: "100x100>" },                     
-                    :path => ":rails_root/public/system/images/news/:id/:style/:filename",
-                    :url => "/system/images/news/:id/:style/:filename"
+  has_attached_file :image,
+                    styles: {original: "4000x4000>", large: "800x800>", small: "250x250>", thumb: "100x100>"},
+                    path: ":rails_root/public/system/images/news/:id/:style/:filename",
+                    url: "/system/images/news/:id/:style/:filename"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.